mobile/src/modules/auth/screens/SendScreen.tsx

```diff
@@ -12,48 +13,72 @@ import { core } from '../../../core/translations';
 import { AuthReduxState } from '../store/reducer';
 
 interface IProps extends InjectedReduxProps {
-  auth: AuthReduxState,
+  auth: AuthReduxState;
 }
+
+interface SendBurstFormState {
+  recipientAddress: string;
+  amountNQT: string;
+  feeNQT: string;
+  showMessage: boolean;
+  advanced: boolean;
+  message: string;
+  encrypt: boolean;
+  deadline: string;
+  pin: string;
+}
+
 type Props = IProps & NavigationInjectedProps;
 
 class Send extends React.PureComponent<Props> {
```

Comment:
we can use `React.PureComponent<Props, State> {...` instead `state: State` somewhere later